changelog/db.changelog-1.31.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.1" encoding="utf-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+<changeSet id="31" author="Ross Jarvis">
+  <createTable tableName="herd">
+    <column name="id" type="uuid" defaultValueComputed="public.uuid_generate_v4()">
+      <constraints nullable="false" primaryKey="true" />
+    </column>
+    <column name="version" type="int">
+      <constraints nullable="false" primaryKey="true" />
+    </column>
+    <column name="applicationReference" type="varchar">
+      <constraints nullable="false" />
+    </column>
+    <column name="herdName" type="varchar" />
```

Comment:
Should some more of these columns be nullable = false? Like the herdname and cph for example, maybe isCurrent, othersOnCph, to be fair maybe all of them?